worlds/keymasters_keep/data/mapping_data.py

```diff
@@ -501,6 +502,109 @@
+    KeymastersKeepRegions.THE_SHADOWY_PASSAGE: KeymastersKeepLocations.THE_SHADOWY_PASSAGE_COMPLETE,
+    KeymastersKeepRegions.THE_SHIMMERING_PASSAGE: KeymastersKeepLocations.THE_SHIMMERING_PASSAGE_COMPLETE,
+    KeymastersKeepRegions.THE_SHROUDED_GATEWAY: KeymastersKeepLocations.THE_SHROUDED_GATEWAY_COMPLETE,
+    KeymastersKeepRegions.THE_SHROUDED_PORTAL: KeymastersKeepLocations.THE_SHROUDED_PORTAL_COMPLETE,
+    KeymastersKeepRegions.THE_SILENT_ARCHWAY: KeymastersKeepLocations.THE_SILENT_ARCHWAY_COMPLETE,
+    KeymastersKeepRegions.THE_SILENT_PASSAGE: KeymastersKeepLocations.THE_SILENT_PASSAGE_COMPLETE,
+    KeymastersKeepRegions.THE_SILENT_THRESHOLD: KeymastersKeepLocations.THE_SILENT_THRESHOLD_COMPLETE,
+    KeymastersKeepRegions.THE_SILENT_VAULT: KeymastersKeepLocations.THE_SILENT_VAULT_COMPLETE,
+    KeymastersKeepRegions.THE_UNFATHOMABLE_DOOR: KeymastersKeepLocations.THE_UNFATHOMABLE_DOOR_COMPLETE,
+    KeymastersKeepRegions.THE_UNKNOWN_ARCH: KeymastersKeepLocations.THE_UNKNOWN_ARCH_COMPLETE,
+    KeymastersKeepRegions.THE_UNKNOWN_GATEWAY: KeymastersKeepLocations.THE_UNKNOWN_GATEWAY_COMPLETE,
+    KeymastersKeepRegions.THE_UNMARKED_PASSAGE: KeymastersKeepLocations.THE_UNMARKED_PASSAGE_COMPLETE,
+    KeymastersKeepRegions.THE_UNMARKED_VAULT: KeymastersKeepLocations.THE_UNMARKED_VAULT_COMPLETE,
+    KeymastersKeepRegions.THE_UNRAVELED_DOOR: KeymastersKeepLocations.THE_UNRAVELED_DOOR_COMPLETE,
+    KeymastersKeepRegions.THE_UNSEEN_ARCHWAY: KeymastersKeepLocations.THE_UNSEEN_ARCHWAY_COMPLETE,
+    KeymastersKeepRegions.THE_UNSEEN_DOOR: KeymastersKeepLocations.THE_UNSEEN_DOOR_COMPLETE,
+    KeymastersKeepRegions.THE_UNSEEN_PASSAGE: KeymastersKeepLocations.THE_UNSEEN_PASSAGE_COMPLETE,
+    KeymastersKeepRegions.THE_UNSEEN_PORTAL: KeymastersKeepLocations.THE_UNSEEN_PORTAL_COMPLETE,
+    KeymastersKeepRegions.THE_UNSPOKEN_GATE: KeymastersKeepLocations.THE_UNSPOKEN_GATE_COMPLETE,
+    KeymastersKeepRegions.THE_UNTOLD_GATEWAY: KeymastersKeepLocations.THE_UNTOLD_GATEWAY_COMPLETE,
+    KeymastersKeepRegions.THE_UNTRACEABLE_PATH: KeymastersKeepLocations.THE_UNTRACEABLE_PATH_COMPLETE,
+    KeymastersKeepRegions.THE_VANISHING_ARCHWAY: KeymastersKeepLocations.THE_VANISHING_ARCHWAY_COMPLETE,
+    KeymastersKeepRegions.THE_VANISHING_DOOR: KeymastersKeepLocations.THE_VANISHING_DOOR_COMPLETE,
+    KeymastersKeepRegions.THE_VAULT_OF_WHISPERS: KeymastersKeepLocations.THE_VAULT_OF_WHISPERS_COMPLETE,
+    KeymastersKeepRegions.THE_VEILED_PASSAGE: KeymastersKeepLocations.THE_VEILED_PASSAGE_COMPLETE,
+    KeymastersKeepRegions.THE_VEILED_PATH: KeymastersKeepLocations.THE_VEILED_PATH_COMPLETE,
+    KeymastersKeepRegions.THE_WHISPERED_PORTAL: KeymastersKeepLocations.THE_WHISPERED_PORTAL_COMPLETE,
+    KeymastersKeepRegions.THE_WHISPERED_THRESHOLD: KeymastersKeepLocations.THE_WHISPERED_THRESHOLD_COMPLETE,
+    KeymastersKeepRegions.THE_WHISPERING_DOOR: KeymastersKeepLocations.THE_WHISPERING_DOOR_COMPLETE,
+}
+
```

Comment:
I just want to say that this diff sucks, and it is the main reason this PR is taking so long to merge